dashboard/test/models/school_test.rb

```diff
@@ -22,6 +27,34 @@ class SchoolTest < ActiveSupport::TestCase
     )
   end
 
+  test 'merge_from_csv in dry run mode with blank table makes no database writes' do
+    # Populate school districts, since schools depends on them as a foreign key.
+    SchoolDistrict.seed_all(stub_school_data: true, force: true)
+
+    School.merge_from_csv(School.get_seed_filename(true), is_dry_run: true)
+    assert_equal 0, School.count
+  end
+
+  test 'merge_from_csv in dry run mode with existing rows makes no database writes' do
+    # Populate school districts, since schools depends on them as a foreign key.
+    SchoolDistrict.seed_all(stub_school_data: true, force: true)
+    School.merge_from_csv(School.get_seed_filename(true))
+
+    # Arbitrary change that should result in an update to each row.
+    parse_row = proc do |row|
+      {
+        id: row['id'],
+        state_school_id: row['state_school_id'],
+        name: row['name'] + 'test'
+      }
+    end
+
+    School.any_instance.expects(:save!).never
```

Comment:
I'd like a more general way of saying "no database activity on Schools table, if that exists. In current form, only explicitly tests the methods that are currently used in `merge_from_csv`, but wouldn't catch, say, the use of `save` (no bang)